src/posts/2024/hello-world.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Hello World!
+teaser: Check out the first post on my new blog!
+tags: javascript, react
+author: "Adrienne Ross"
+date: "2019-07-10T16:04:44.000Z"
+---
+
+Hello, world! This is my new markdown blog created with Next.js.
+
+Right now, I plan to display blog posts from local Markdown files in the repo. In the future, I will fetch my Markdown posts from TinaCMS.
```

Comment:
💭 Do we want to to link to Tina CMS? Also, why would you choose Tina CMS? Maybe it could be a later goal to re-write this first blog post, and clearly state my intentions as opposed to speaking towards the tech stack of the project without any context as to why I made those choices.

Reply:
Let's do this lata allagata